.github/actions/set-env-variables/action.yml

```diff
@@ -15,3 +15,4 @@ runs:
         CILIUM_CLI_VERSION="v0.15.20"
         echo "CILIUM_CLI_VERSION=$CILIUM_CLI_VERSION" >> $GITHUB_ENV
         echo "PUSH_TO_DOCKER_HUB=true" >> $GITHUB_ENV
+        echo "GCP_PERF_RESULTS_BUCKET=gs://cilium-perf-results" >> $GITHUB_ENV
```

Comment:
Can we remove this variable from the other scale workflows? They still set it manually: https://github.com/cilium/cilium/blob/7fcdb47469dc4c7f4326284404913efecad6b065/.github/workflows/scale-test-100-gce.yaml#L39

Reply:
Good idea, done.